tests/test_lpp.py

```diff
@@ -29,11 +29,12 @@ def test_one_lpp():
     q = 4
 
     # Generic function that runs lightkurve and returns a lightkurve object
-    lcf = search_lightcurvefile(
-        target_name, quarter=q, mission=mission).download()
-    lc = lcf.SAP_FLUX.remove_nans().remove_outliers()
+    lcf = search_lightcurve(
+        target_name, quarter=q, mission=mission, exptime=1800).download(
+            flux_column="sap_flux")
+    lc = lcf.remove_nans().remove_outliers()
     flat = lc.flatten(window_length=81)
-    flat.flux = flat.flux - 1.0
+    flat.flux = flat.flux.value - 1.0
```

Comment:
I am not sure if this is completely correct fix, though it does make the test pass again.

Reply:
I guess ideally we would like to make sure that flat.flux persists as a quantiity instead of as a float. So I think this line should be
`from astropy import units as u`
`flat.flux = flat.flux.value - 1.0) * u.electron/u.electron`
The units will be dimensionless, there may be a better way to assign dimensionless units.
(The units returned by lk.flatten are actually incorrect.)

Reply:
Does the math of minus 1 e and then divided by 1 e make sense? What if somehow another input has flux in Jy? Then, what?

Reply:
In the tag up today, Susan and I agreed to leave this as-is for now as not to change the currently behavior.